gulpfile.js

```diff
@@ -27,10 +27,10 @@ var releaseDir = './release/';
 
 // Get platform from commandline args
 // #
-// # gulp <task> [<platform>]+        Run only for platform(s) (with <platform> one of --linux64, --linux32, --osx64, or --win32 --chromeos)
+// # gulp <task> [<platform>]+        Run only for platform(s) (with <platform> one of --linux64, --linux32, --osx64, --osx32, or --win32 --chromeos)
```

Comment:
I forgot to add `--win64` when I did the PR. I don't know if it can be included here or is better to make a PR only to change this comment.